hbase-common/src/main/java/org/apache/hadoop/hbase/util/Bytes.java

```diff
@@ -58,6 +58,7 @@
 @edu.umd.cs.findbugs.annotations.SuppressWarnings(
     value = "EQ_CHECK_FOR_OPERAND_NOT_COMPATIBLE_WITH_THIS",
     justification = "It has been like this forever")
+@SuppressWarnings("MixedMutabilityReturnType")
```

Comment:
> This method returns both mutable and immutable collections or maps from different paths. This may be confusing for users of the method.

Yeah, that's a fair description.